optimism/src/mips/witness.rs

```diff
@@ -211,6 +220,12 @@ impl<Fp: Field> Env<Fp> {
         }
     }
 
+    pub fn write_column(&mut self, column: Column, value: u64) {
+        match column {
+            Column::ScratchState(idx) => self.scratch_state[idx] = value.into(),
+        }
```

Comment:
I guess as soon as we add more variants into Column then we will have to add the branch `_ => panic!("Unexpected column"),` in the match, as in the demo?